src/main/java/com/example/bookstorewebapp/exception/CustomGlobalExceptionHandler.java

```diff
@@ -57,6 +60,7 @@ private ResponseEntity<Object> getObjectResponseEntity(String message, Exception
         body.put(TIMESTAMP, LocalDateTime.now());
         body.put(STATUS, HttpStatus.INTERNAL_SERVER_ERROR);
         body.put(ERROR, message);
+        log.error("Got an error !!! " + ex);
```

Comment:
Usually we don't need to log client errors cause they are not so useful and occupy a lot of memory. Log only undandled exceptions (programmer errors) and pass exception to the log too.
```suggestion
        log.error(ex.getMessage(), ex);
```